cinema/tests/test_movie_api.py

```diff
@@ -43,9 +43,7 @@ def sample_actor(**params):
 
 
 def sample_movie_session(**params):
-    cinema_hall = CinemaHall.objects.create(
-        name="Blue", rows=20, seats_in_row=20
-    )
+    cinema_hall = CinemaHall.objects.create(name="Blue", rows=20, seats_in_row=20)
```

Comment:
please, if you are using black - use it on needed files. Stop using it on the whole project, because you don't have to change or even reformat files that was not made or implemented by you)